app/src/main/java/org/listenbrainz/android/di/ServiceModule.kt

```diff
@@ -12,31 +12,50 @@ import dagger.hilt.components.SingletonComponent
 import org.listenbrainz.android.model.yimdata.YimData
 import org.listenbrainz.android.service.BlogService
 import org.listenbrainz.android.service.ListensService
+import org.listenbrainz.android.service.SocialService
 import org.listenbrainz.android.service.YimService
 import org.listenbrainz.android.util.Constants.LISTENBRAINZ_API_BASE_URL
+import org.listenbrainz.android.util.Constants.MUSICBRAINZ_AUTH_BASE_URL
 import retrofit2.Retrofit
 import retrofit2.converter.gson.GsonConverterFactory
 import java.lang.reflect.Type
 import javax.inject.Singleton
 
-@Module
+@Module(includes = [AppModule::class])
 @InstallIn(SingletonComponent::class)
 class ServiceModule {
     
+    /** Retrofit instance with LB api url.*/
+    private val retrofitBuilderLB: Retrofit = Retrofit.Builder()
+        .baseUrl(LISTENBRAINZ_API_BASE_URL)
+        .addConverterFactory(GsonConverterFactory.create())
+        .build()
+    
+    /** Retrofit instance with MB api url.*/
+    private val retrofitBuilderMB: Retrofit = Retrofit.Builder()
```

Comment:
We don't need MB anymore. Please remove.

Reply:
Alright